.github/workflows/build.yml

```diff
@@ -105,6 +105,30 @@ jobs:
           name: wheel-${{ matrix.name }}-${{ matrix.cpython }}-${{ matrix.architecture }}
           path: dist/*.whl
 
-      - name: "Lint"
+
+  quality:
```

Comment:
How it is supposed to work?
I do not see anything about Quality in the github actions logs.

Reply:
I'm looking at the checks listed below; it's appearing near the end of the list on my display:

> ![image](https://github.com/user-attachments/assets/af38b7bb-87b0-4e60-ab72-99824aad539e)